### PR TITLE
app-text/mupdf: Usage of environment variables, die calls

### PR DESCRIPTION
**Pull request by rndxelement**

### CODE REVIEW

File: app-text/mupdf/mupdf-1.19.0.ebuild [hunk 1]
```diff
@@ -122,32 +122,32 @@ src_install() {
 		domenu platform/debian/${PN}.desktop
 		doicon platform/debian/${PN}.xpm
 	else
-		rm docs/man/${PN}.1 || die
+		rm docs/man/"${PN}".1 || die "Failed to rm in src_install()"
```
Comment (thesamesam) [inline app-text/mupdf/mupdf-1.19.0.ebuild:125]: Note that ${PN} can't contain spaces, so I usually wouldn't bother quoting it.

I'd say it arguably looks cleaner to do the whole string if you're so inclined, or just leave it.

Comment (thesamesam) [reply app-text/mupdf/mupdf-1.19.0.ebuild:125]: On the die message, maybe make it more specific ("couldn't cleanup man page") or similar? same applies to others

File: app-text/mupdf/mupdf-1.19.0.ebuild [hunk 2]
```diff
@@ -122,32 +122,32 @@ src_install() {
...
+		dosym "${PN}"-gl /usr/bin/"${PN}"
 	elif use X ; then
 		einfo "mupdf symlink points to mupdf-x11 (bug 616654)"
-		dosym ${PN}-x11 /usr/bin/${PN}
+		dosym "${PN}"-x11 /usr/bin/"${PN}"
 	fi
 
 	# Respect libdir (bug #734898)
-	sed -i -e "s:/lib:/$(get_libdir):" platform/debian/${PN}.pc || die
+	sed -i -e "s:/lib:/$(get_libdir):" platform/debian/${PN}.pc || die "Failed to sed pkgconfig file to respect libdir in src_install()"
```
Comment (thesamesam) [inline app-text/mupdf/mupdf-1.19.0.ebuild:147]: nice description, although I think the error message will mention the phase

File: app-text/mupdf/mupdf-1.19.0.ebuild [hunk 3]
```diff
@@ -122,32 +122,32 @@ src_install() {
...
 		-e "1iprefix = ${ED}/usr" \
 		-e "1ilibdir = ${ED}/usr/$(get_libdir)" \
 		-e "1idocdir = ${ED}/usr/share/doc/${PF}" \
-		-i Makerules || die
+		-i Makerules || die "Failed to sed Makerules in src_install()"
 
 	_emake install
 
-	dosym libmupdf.so.${PV} /usr/$(get_libdir)/lib${PN}.so
+	dosym libmupdf.so."${PV}" /usr/$(get_libdir)/lib"${PN}".so
```
Comment (thesamesam) [inline app-text/mupdf/mupdf-1.19.0.ebuild:136]: PV + PN can't contain spaces

**Comment by rndxelement** [comment]
Thanks @thesamesam  for the super fast review. I edited the die messages again to make them more specific.

**Pull request closed without merging**